lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1363,6 +1363,14 @@ void request_evaluate(const llvm::json::Object &request) {
   lldb::SBFrame frame = g_dap.GetLLDBFrame(*arguments);
   std::string expression = GetString(arguments, "expression").str();
   llvm::StringRef context = GetString(arguments, "context");
+  static std::string last_nonempty_expression;
```

Comment:
Do not use static object. Move to DAP object instead.

Reply:
Done.